fleetdm/fleet
# Fixed uncaught exception causing 500 popover

Relates to #2924 

Fixed uncaught exception from Host Details causing 500 popover.

# Checklist for submitter

If some of the following don't apply, delete the relevant line.

- [ ] ~Changes file added (for user-visible changes)~
- [ ] ~Documented any API changes (docs/01-Using-Fleet/03-REST-API.md)~
- [ ] ~Documented any permissions changes~

# QA Steps
- Open 2 browsers (or 1 regular tab and 1 incognito tab)
- On the 1st, log in as an admin
- On the 2nd, log in as another user, then navigate to the Manage Hosts page
- As the admin, navigate to Settings >> Users
- As the admin, reset the sessions of the other user in which you logged in
- Once you received confirmation, go back to the other user's tab
- As the other user, click on a host from the table
- Verify you do not receive the 500 page
